api/user.go

```diff
@@ -13,6 +13,17 @@ import (
 	"github.com/miniflux/miniflux/http/response/json"
 )
 
+
+// CurrentUser is the API handler to retrieve the user id
+func (c *Controller) CurrentUser(w http.ResponseWriter, r *http.Request) {
+	ctx := context.New(r)
+	user, err := c.store.UserByID(ctx.UserID())
+	if err != nil {
+		json.Forbidden(w)
```

Comment:
Thank you for addressing my comments, but a `return` statement still missing here.

Reply:
Hahaha, sorry. I knew I should have tested it instead of cowboy coding!